frontend/src/app/applications/temporary-outfitters/temporary-outfitters.component.ts

```diff
@@ -116,6 +116,16 @@ export class TemporaryOutfittersComponent implements DoCheck {
     });
   }
 
+  matchUrls(): void {
```

Comment:
In our sprint planning discussion, we said that we were not interested in #791 right now. It looks like this code was committed and despite us having identified a bug at sprint review and saying that we were not interested in incomplete stories getting added to the code. For things like this in the future, I would rather we revert the code.